scripts/auth.js

```diff
@@ -0,0 +1,45 @@
+//static JSON array for users
+const users = [
+    { username: "Abd-ElSalam", password: "Abd-ElSalam" },
+    { username: "Emara", password: "Emara" },
+    { username: "Ramadan", password: "Ramadan" },
+    { username: "Hany", password: "Hany" },
+    { username: "Kiro", password: "Kiro" },
+    { username: "Hussein", password: "Hussein" }
+];
+
+//add event listner when page is loaded
```

Comment:
```suggestion
//add event listener when page is loaded
```
A little typo in the comment.